source/code/plugins/filter_auditd_plugin.rb

```diff
@@ -32,4 +32,36 @@ def filter(tag, time, record)
         return audit_blob
     end # filter
   end # class
+
+  class AuditdDSCLogFilter < Filter
+
+    Fluent::Plugin.register_filter('filter_auditd_dsc_log', self)
```

Comment:
where's the `filter_auditd_dsc_log` being used? don't see in any .conf

Reply:
It is used in the auditd_plugin.conf. See: dsc/Providers/Modules/nxOMSAuditdPlugin/conf/auditd_plugin.conf
Here is the relevent PR in the dsc repo: https://github.com/microsoft/PowerShell-DSC-for-Linux/pull/758